SPECS/moby-containerd-cc/moby-containerd-cc.spec

```diff
@@ -0,0 +1,84 @@
+%global debug_package %{nil}
+%define upstream_name containerd-cc
+%define upstream_repo confidential-containers-containerd
+%define commit_hash 4a2809f776500dfb8e4ed33db7f4e05ed68edfbf
+
+Summary: Industry-standard container runtime for confidential containers
+Name: moby-%{upstream_name}
+Version: 1.7.0
+Release: 1%{?dist}
+License: ASL 2.0
+Group: Tools/Container
+URL: https://www.containerd.io
+Vendor: Microsoft Corporation
+Distribution: Mariner
+
+Source0:  https://github.com/microsoft/confidential-containers-containerd/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1: containerd.service
+Source2: containerd.toml
+
+%{?systemd_requires}
+
+BuildRequires: btrfs-progs-devel
+BuildRequires: git
+BuildRequires: golang
+BuildRequires: go-md2man
+BuildRequires: make
+BuildRequires: systemd-rpm-macros
+
+Requires: moby-runc >= 1.1.0
+
+Conflicts: containerd
```

Comment:
is containerd aliased with moby-containerd or did we mean moby-containerd here?

Reply:
I think this is referring to the binary. It's the same in the base moby-containerd spec

Reply:
It's possible the current spec is outdated with its `Conflicts`. This now conflicts with moby-containerd